src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  return (
+    sourceString
+      // Breaks sourceString by semicolon into separate stings
+      .split(';')
```

Comment:
It's good practice to trim the string before splitting it to handle cases where the string might start with whitespace or semicolons.